mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/154 giant computers

To test:

- open a scenario that has computers you can buy
- purchase the Blato Desktop Select
- Notice the computer should be a more reasonable size

Closes #154
